lua/nvim-web-devicons/icons-default.lua

```diff
@@ -107,6 +107,12 @@ local icons_by_filename = {
     cterm_color = "28",
     name = "Gvimrc",
   },
+  [".justfile"] = {
+    icon = "",
```

Comment:
Is that a just specific icon?

If not can we please use the general `"config"` icon ``

Reply:
I simply used the icon that `eza` is using for Justfiles. That airtight it should I change it?

Reply:
Is it [eza](https://github.com/eza-community/eza) you are referring you to?

If so, how do we see their icon (source)?

Reply:
Yes. I think this is the location in their source code: https://github.com/eza-community/eza/blob/main/src/output/icons.rs#L281C4-L281C57

and in the terminal, shows like this: 
![Pasted image 2024-06-07 at 02 01 04@2x](https://github.com/nvim-tree/nvim-web-devicons/assets/73286100/d2023d96-dd7e-442d-bbf8-c39b93173ee8)